src/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './components/ReactSlider/ReactSlider';
```

Comment:
Since we only have one export, we should probably just point directly to ReactSlider.